File: backend/micro_epoweredmove_paymentmethod/src/main/java/com/kosdiam/epoweredmove/models/PaymentMethod.java
package com.kosdiam.epoweredmove.models;

import java.io.Serializable;

import com.kosdiam.epoweredmove.models.enums.PaymentType;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class PaymentMethod implements Serializable {
    private String id;
    private PaymentType description;
    private String poiId;
}

File: frontend/E-PoweredMoveAndroidApp/app/src/main/java/com/kosdiam/epoweredmove/models/PaymentMethod.kt
package com.kosdiam.epoweredmove.models

import android.os.Parcelable
import com.kosdiam.epoweredmove.models.enums.PaymentType
import kotlinx.android.parcel.Parcelize

@Parcelize
data class PaymentMethod(
    val id: String,
    val description: PaymentType,
    val poiId: String,
) : Parcelable
